src/Components/Patient/DailyRounds.tsx

```diff
@@ -598,8 +536,7 @@ export const DailyRounds = (props: any) => {
                 [
                   "investigations",
                   "icd11_diagnosis",
-                  "prescriptions",
-                  "prn_prescriptions",
+                  "additional_symptoms",
```

Comment:
This field is no longer used and has been removed in https://github.com/ohcnetwork/care/pull/2410

```suggestion
```

Reply:
I am using the key just to store symptom info for scribe use, it is not relevant for sending to the backend. It will not be a problem if it is removed.